indico/modules/events/registration/fields/simple.py

```diff
@@ -317,8 +318,12 @@ class CountryField(RegistrationFormFieldBase):
 
     @classmethod
     def unprocess_field_data(cls, versioned_data, unversioned_data):
+        def unaccent_cmp(x, y):
+            return locale.strcoll(x['caption'], y['caption'])
```

Comment:
```suggestion
        def unaccent_cmp(a, b):
            return locale.strcoll(a['caption'], b['caption'])
```

this is more common in comparison funcs than x/y